dts/msc-sm2-imx6x.dtsi

```diff
@@ -648,6 +648,11 @@
 	clock-frequency = <100000>;
```

Comment:
Commit message should be < 50 chars
And it would be good to describe what this is and what information it contains for reference.